nano/node/transport/channel.cpp

```diff
@@ -16,28 +16,24 @@ nano::transport::channel::channel (nano::node & node_a) :
 
 void nano::transport::channel::send (nano::message & message_a, std::function<void (boost::system::error_code const &, std::size_t)> const & callback_a, nano::transport::buffer_drop_policy drop_policy_a, nano::transport::traffic_type traffic_type)
 {
-	auto buffer (message_a.to_shared_const_buffer ());
-	auto detail = to_stat_detail (message_a.header.type);
-	auto is_droppable_by_limiter = (drop_policy_a == nano::transport::buffer_drop_policy::limiter);
-	auto should_pass (node.outbound_limiter.should_pass (buffer.size (), to_bandwidth_limit_type (traffic_type)));
-	if (!is_droppable_by_limiter || should_pass)
-	{
-		node.stats.inc (nano::stat::type::message, detail, nano::stat::dir::out);
-		node.logger.trace (nano::log::type::channel_sent, nano::to_log_detail (message_a.type ()),
-		nano::log::arg{ "message", message_a },
-		nano::log::arg{ "channel", *this },
-		nano::log::arg{ "dropped", false });
+	auto buffer = message_a.to_shared_const_buffer ();
+
+	bool is_droppable_by_limiter = (drop_policy_a == nano::transport::buffer_drop_policy::limiter);
+	bool should_pass = node.outbound_limiter.should_pass (buffer.size (), to_bandwidth_limit_type (traffic_type));
```

Comment:
If it is not droppable by limiter then we don't want to evaluate should_pass.

Reply:
The code that was there before:
```
auto is_droppable_by_limiter = (drop_policy_a == nano::transport::buffer_drop_policy::limiter);
	auto should_pass (node.outbound_limiter.should_pass (buffer.size (), to_bandwidth_limit_type (traffic_type)));
	if (!is_droppable_by_limiter || should_pass)
	{
```

Reply:
> If it is not droppable by limiter then we don't want to evaluate should_pass.
But I also don't think it's a good idea to change it, it would lead to some strange and undesirable behavior.

Reply:
Ok